benches/suites/progress_set.rs

```diff
@@ -146,14 +147,32 @@ pub fn bench_progress_set_iter(c: &mut Criterion) {
     });
 }
 
-pub fn bench_progress_set_nodes(c: &mut Criterion) {
+pub fn bench_progress_set_voters(c: &mut Criterion) {
     let bench = |voters, learners| {
         move |b: &mut Bencher| {
             let set = quick_progress_set(voters, learners);
             b.iter(|| {
                 let set = set.clone();
-                let agg = set.iter().all(|_| true);
-                agg
+                set.voters().for_each(|_| {});
```

Comment:
This line looks like the kind of code that could be optimized away completely. Are you confident it's not?